### PR TITLE
Clarify that a platform MUST include operation in polling

### PR DESCRIPTION
**Pull request by duglin**
See: https://github.com/openservicebrokerapi/servicebroker/pull/166/files#r113095299

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by angarg12** [comment]
LGTM

File: spec.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ The request provides these query string parameters as useful hints for brokers.
 |---|---|---|
 | service_id  |  string | ID of the service from the catalog.  |
 | plan_id  | string  | ID of the plan from the catalog.  |
-| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client SHOULD provide the same value using this query parameter as a URL-encoded string.  |
+| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client MUST provide the same value using this query parameter as a URL-encoded string.  |
```
Comment (avade) [inline spec.md]: Do we define what we mean by "broker client" anywhere in the spec?  It's technically correct, but we often refer to this in our discussions as the platform.

Comment (duglin) [inline spec.md]: I was going to open a PR to add a "terminology" section - and then force us to use the right terms.  See https://github.com/openservicebrokerapi/servicebroker/pull/179 where I s/broker client/platform/g

**Comment by angarg12** [comment]
lgtm

**Comment by duglin** [comment]
LGTM

**Comment by avade** [comment]
LGTM

**Comment by avade** [comment]
@angarg12 you need to uppercase your LTGM

**Comment by duglin** [comment]
actually, he had already LGTM'd that's why it was ignored

**Comment by angarg12** [comment]
oops, my bad, i thought we needed another round.

also my keyboard broke and i can't uppercase.

**Comment by vaikas-google** [comment]
LGTM

**Pull request merged**